README.md

```diff
@@ -139,6 +153,99 @@ You can find an example of the SQL file at `tests/sql_script.sql`.
 
 Default: `null`
 
+Type: `str`
+
+### `mssql_enable_sql_agent`
+
+Set this variable to `true` or `false` to enable or disable the SQL agent.
+
+Default: `null`
+
+Type: `bool`
+
+### `mssql_install_fts`
+
+Set this variable to `true` or `false` to install or remove the
+`mssql-server-fts` package that provides full-text search.
+
+Default: `null`
+
+Type: `bool`
+
+### `mssql_enable_ha`
+
+Set this variable to `true` or `false` to install or remove the
+`mssql-server-ha` package and enable or disable the `hadrenabled` setting.
+
+Default: `null`
+
+Type: `bool`
+
+### `mssql_tune_for_fua_storage`
+
+Set this variable to `true` or `false` to enable or disable settings that
+improve performance on hosts that support Forced Unit Access (FUA) capability.
+
+Only set this variable to `true` if your hosts are configured for FUA
+capability.
+
+When set to `true`, the role applies the following settings:
+
+* Set the `traceflag 3979 on` setting to enable trace flag 3979 as a startup
+parameter
+* Set the `control.alternatewritethrough` setting to `0`
+* Set the `control.writethrough` setting to `1`
+
+When set to `false`, the role applies the following settings:
+
+* Set the `traceflag 3982 off` parameter to disable trace flag 3979 as a
+startup parameter
+* Set the `control.alternatewritethrough` setting to its default value `0`
+* Set the `control.writethrough` setting to its default value `0`
+
+For more details, see SQL Server and Forced Unit Access (FUA) I/O subsystem
+capability at <https://docs.microsoft.com/en-us/sql/linux/sql-server-linux-performance-best-practices?view=sql-server-ver15>.
+
+Default: `null`
+
+Type: `bool`
+
+### `mssql_enable_tls_encryption`
+
+Use this variable to configure MSSQL to encrypt connections using TLS
+certificates. You must have the TLS certificate and private key on the Ansible
+control node.
+
+When you use this variable, the role copies the TLS certificate and private key
+files to MSSQL and configure MSSQL to use these files to encrypt connections.
+
+Default: `null`
+
+Type: `dict`
+
+#### `certificate`
```

Comment:
For the crypto parameter names, please use the conventions here - https://linux-system-roles.github.io/documentation/tls_crypto_parameter_and_key_names.html

if `certificate` is the name of an x509 certificate file on the controller to copy to the managed host, use `cert_src`

also - `private_key_src`

Reply:
Done.